spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/SparkTestHelperBase.java

```diff
@@ -68,7 +68,7 @@ protected void assertEquals(
     }
   }
 
-  private void assertEquals(String context, Object[] expectedRow, Object[] actualRow) {
+  protected void assertEquals(String context, Object[] expectedRow, Object[] actualRow) {
```

Comment:
I don't think this get's overridden now?

Reply:
it doesn't get overriden but we use it in the test